test/studies/hpcc/common/bradc/Block1D.chpl

```diff
@@ -533,12 +533,12 @@ class LocBlock1DDom {
   // this is the parallel iterator for the local domain, see global
   // domain parallel iterators for general notes on the approach
   //
-  proc these(param tag: iterKind) where tag == iterKind.leader {
+  iter these(param tag: iterKind) where tag == iterKind.leader {
     halt("This is bogus");
     yield {1..100};
```

Comment:
Optional: declare the return type of the above iterator as `: domain(1)` (goes before `where`). Otherwise it will yield `none` (I think).